pom.xml

```diff
@@ -1392,10 +1376,11 @@
       </dependency>
 
       <dependency>
-        <groupId>xom</groupId>
+        <!-- New group Id -->
+        <groupId>com.io7m.xom</groupId>
         <artifactId>xom</artifactId>
```

Comment:
We should probably add xom:xom as a banned dependency, to make sure we always use the right one.